query-service-impl/src/main/java/org/hypertrace/core/query/service/QueryFunctionConstants.java

```diff
@@ -18,4 +18,5 @@ public interface QueryFunctionConstants {
   String QUERY_FUNCTION_STRINGEQUALS = "STRINGEQUALS";
   String QUERY_FUNCTION_CONDITIONAL = "CONDITIONAL";
   String QUERY_FUNCTION_CONCAT_OR_NULL = "CONCATORNULL";
+  String DATA_TIME_CONVERT = "DATETIMECONVERT";
```

Comment:
Q: we will define this function at Postgres side?

Reply:
Yes, we will define it on postgres side